src/lib/store/actions.js

```diff
@@ -34,9 +34,12 @@ export function fetchTxInfo () {
     dispatch(update('txGasEstimate', gasEstimate));
     dispatch(update('txGasPrice', gasPrice));
     dispatch(update('txBalance', balance));
-    dispatch(update('txValue', params.value));
-  }
 
+    dispatch(update('txValue', params.value || 0));
+    dispatch(update('txURL', first.url || 'unknown'));
```

Comment:
If I don't escape then the below error is logged everytime (this happens also with `value` field prior to this pull request):

> Uncaught (in promise) Error: Given action "UPDATE", reducer "origin" returned undefined. To ignore an action, you must explicitly return the previous state. If you want this reducer to hold no value, you can return null instead of undefined.

Reply:
To avoid business logic, I instead escaped to null

Reply:
Yes, redux doesn't like to return `undefined` state. This can be mitigated in reducer itself.

```
  if (action.type == 'UPDATE' && action.prop == 'txOrigin') {
    return action.value || null;
```

and the're no mixing :wink: